README.md

```diff
@@ -365,6 +365,49 @@ private Binding<HttpStatus> problems(final Condition<HttpStatus> condition) {
 }
 ```
 
+#### Template Routing
+
+Routing and response handling is sometimes pretty similar for many request, and can even be handle using the same
+defined routing logic for different requests. To enable reuse of this routing logic, Riptide supports the creation
+of nested *template routing*, that can be reused for dispatching different request. The following example defines
+a nested hierarchy of routing templates:
+
+```java
+static final Router<MediaType> CONTENT_ROUTER = Router.create(contentType(),
+        on(PERTNER).capture(Partner.class),
+        on(CONTRACT).capture(Contract.class),
+        on(PROBLEM).call(Problem.class, Example::propagate),
+        on(ERROR).call(Error.class, Example::propagate),
+        anyContentType().call(Example::failContent));
+
+static final Router<HttpStatus> STATUS_ROUTER = Router.create(status(),
+        on(OK).dispatch(CONTENT_ROUTER),
+        anyStatus().call(Example::failStatus));
+
+static final Router<HttpStatus.Series> SERIES_ROUTER = Router.create(series(),
```

Comment:
Assuming that the gateway who holds those routes is a singleton they don't need to be static, but could be instance fields. That would solve the problem with the instance access as well.